cspm/admin-guide/connect-your-cloud-platform-to-prisma-cloud/onboard-your-oci-account/add-oci-tenant-to-prisma-cloud.adoc

```diff
@@ -37,19 +37,19 @@ image::oci-tenant-console-1.png[scale=40]
 
 .. Return to the Prisma Cloud Onboarding Setup page and paste the OCID in the *Tenant/Root OCID* field.
 
-.. Select the *Home Region* where the tenant is created (for example, us-phoenix-1).
+.. Select the *Home Region* where the tenant is created (for example, us-phoenix-1) and click *Next*.
 
-.. On clicking *Next* and following the steps listed in *Create a User to Enable Access*, a new user, group, and policy that correspond to OCI Identity User Name, Group Name, and Policy Name will be created.
+.. Follow the steps in xref:id5ac2883d-d1ed-44a3-bd63-cc3fabedf477/create-a-user[Create a User to Enable Access], to create a new user, group, and policy that correspond to OCI Identity User Name, Group Name, and Policy Name.
```

Comment:
@Aishbn, please confirm that this xref works.

Reply:
The xref works.